### ISSUE TITLE
Is it ready to be compiled in an actual iOS9 app ?

### ISSUE THREAD

**Issue by numerized** [driftyco/ionic-conference-app#48]
Hi Folks ! Thanks for the conf today was awesome.
I've actually loved the schedule app as a proof of concept.

Is it ready to be compiled in an actual iOS9 app ? It seems that after adding platform ios, building it and running it, an older version shows without maps in a iOS9 device, without speakers list neither and with only a breakfast entry in the schedule tab.

The ionic serve command works perfectly.
Do I miss something ?

**Comment by jbavari** [comment]
@numerized - That is currently an issue we are working through.

Currently, we are using the `DatePipe` to format dates shown in the schedule.

According to the Angular.io documentation about the [DatePipe](https://angular.io/docs/ts/latest/api/core/DatePipe-class.html), it is only usable in Chrome and Opera, unfortunately. 

From the docs:

> WARNING: this pipe uses the Internationalization API. Therefore it is only reliable in Chrome and Opera browsers.
> Formats a date value to a string based on the requested format.

We are currently working on [the issue](https://github.com/driftyco/ionic-conference-app/issues/42). We have outlined a temporary fix in the meantime to work around using DatePipes.

Stay tuned!

**Comment by brandyscarney** [comment]
Hey, I removed the `DatePipe` from the code and reverted back to using a custom `date-format` Component. The issue Josh mentioned explains it more and links to the original issues where we converted to using the `DatePipe`. The app is now working on iOS 9, but we found some new issues with iOS 8.

I submitted an issue for iOS 8 here: https://github.com/driftyco/ionic-conference-app/issues/49

The work around is to not use a `<form>` to wrap the `<ion-segment>` but then `<ion-segment>` loses it's functionality. We will be working on a fix for this.
